tests/constants/data.constants.ts

```diff
@@ -0,0 +1,53 @@
+
+export const csvString = `
+role,name,team
+student,JunWei96,CS2103T-W12-3
+admin,damithc,
+tutor,anubh-v,CS2103T-W12-3
+admin,geshuming,
+tutor,jj-lim,CS2103T-W12-3
+tutor,jj-lim,CS2103T-W12-3
```

Comment:
Line 9 is a copy of line 8; it's kept here since the csv parser is expected to handle redundant lines